handlers/handlers.go

```diff
@@ -166,20 +186,22 @@ func HandleSessionRequest(
 		}
 
 		responseWriter.WriteHeader(http.StatusCreated)
-		fmt.Fprint(responseWriter, "passward email combo was correct")
+		fmt.Fprint(responseWriter, "password email combo was correct")
```

Comment:
are there any conventions of what non web-page api's should return?

Reply:
what do you mean by non web-page api?

Reply:
I consider non web-page as something that's doesn't return a html DOM object like `<html><body><div>lama</div></body></html>`

Reply:
the convention is for them to return whatever data is being asked for or a status code explaining why it couldn't get the data. So in this case, we really should just delete this line